nemo/collections/asr/parts/submodules/rnnt_greedy_decoding.py

```diff
@@ -558,7 +560,13 @@ def __init__(
 
         # Depending on availability of `blank_as_pad` support
         # switch between more efficient batch decoding technique
-        if self.decoder.blank_as_pad:
+
+        if cuda_optimized:
+            if isinstance(decoder_model, StatelessTransducerDecoder):
```

Comment:
Dont import a class like this at the top. Import inside of this if clause instead - otherwise circular import issues may arise later. Also, i don't think `RNNTDecoder` is used.